src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,34 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  onHandleCloseModal: () => void;
+  selectedTodo: Todo | null;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  onHandleCloseModal = () => {},
+  selectedTodo,
+}) => {
+  const [user, setUser] = useState<User>();
+
+  useEffect(() => {
+    if (selectedTodo) {
+      getUser(selectedTodo.userId).then(setUser);
+    }
+  }, []);
 
-export const TodoModal: React.FC = () => {
   return (
-    <div className="modal is-active" data-cy="modal">
+    <div
+      className="modal is-active"
+      data-cy="modal"
+    >
       <div className="modal-background" />
 
-      {true ? (
+      {!user ? (
```

Comment:
Add state for handling `isLoading`, the same as you did it in `App` component

Reply:
> Add state for handling `isLoading`, the same as you did it in `App` component

Still not fixed